src/main/java/drivers/BrowserType.java

```diff
@@ -0,0 +1,9 @@
+package drivers;
+
+public enum BrowserType {
+    FIREFOX,
+    CHROME,
+    EDGE,
+    OPERA
+}
+
```

Comment:
Please use the same code style for all the classes. Part of classes end with space line and part - not

Reply:
done